### DISCUSSION TITLE
How do you edit the code for .dib files without disabling the Interactive Notebooks extension in VSCode?

### DISCUSSION THREAD

**Discussion by kylecurtis** [dotnet/interactive#2027] [Q&A]
I want to edit the code for .dib files, but right now the only way I have found to do so is by opening 2 different VSCode windows, One window with the .NET Interactive Notebooks extension enabled, and the other one without it enabled. 

For the window that has the extension disabled, the code appears like I want it to. For example:

```
#!csharp
Console.WriteLine("Hello, World!");
```

And when I edit this markdown code it changes for the other window which displays the code and allows me to run it.

Is there an extension setting to allow me to do all of my development using code instead of using all of the cell buttons and UI?

I'm thinking along the lines of markdown files. How you can edit the markdown code in one tab and split the editor to display the final result.

**Comment by dfinke** [comment]
I right mouse click the file, choose `Open With...', then select `Text Editor`. Don't know if there is a setting to make that the default.

**Reply by kylecurtis** [reply]
I can do the same with other editors and it even displays the file's code on GitHub but I just cant get it to display in Visual Studio Code without disabling the .NET Interactive Notebooks extension. Do you think it's worth submitting a request for it? I personally think it would a useful feature to have.

**Reply by jonsequitur** [reply]
You can use the `Reopen editor with...` command to switch to plain text editing mode.

![image](https://user-images.githubusercontent.com/547415/173923874-b49c3a01-fee4-4fb3-b891-ec0aa3975366.png)
